### DISCUSSION TITLE
How to say no to a GitHub issue feature request?

### DISCUSSION THREAD

**Discussion by scottfurry** [sqlitebrowser/sqlitebrowser#3129] [General]
I encountered this [thread of discussion](https://news.ycombinator.com/item?id=32366248) today. After reading through the different posts, I could see that it contained a few little nuggets of wisdom that maintainers would find useful.

Posting for awareness and general reading.

**Comment by justinclift** [comment]
Thanks.  Saw that the other day, though fortunately we don't get many people who are unreasonably demanding and need telling to go jump. :smile:

**Reply by scottfurry** [reply]
...or otherwise study their aerodynamic suitability from tall structures.
